automatic timelock duration for decrease timelock (#722)
morpho-org/vault-v2: README.md

If a gate is not set, its corresponding operations are not restricted.

Gate changes can be timelocked.

> By setting the timelock to `type(uint256).max`, a curator can commit to an irreversible gate setup.

Author: thought the same but I feel it's not worth the added code, wdyt?

> By setting the timelock an extremely high value such as `2**256-1`, a curator can irreversibly commit to a gate setup.

Reviewer: ```suggestion
By setting the timelock to `type(uint256).max`, a curator can irreversibly commit to a gate setup.
```

The curator's role is to curate the vault, meaning setting risk limits, gates, allocators, fees.
Only one address can have this role.

> Some actions of the curator are timelockable.

Reviewer: ```suggestion
Curator actions that can safely be timelocked are timelockable.
```

> Curator actions that can safely be timelocked are timelockable.

Reviewer: actually I don't like it anymore because some things need to be timelocked in order to get the vault's properties 
```suggestion
Curator actions are timelockable, except the ones that cannot safely be.
```

- Decrease relative caps.
- [Timelockable] Set adapters.
- [Timelockable] Set allocators.
- [Timelockable] Increase timelocks.
- [Timelocked by the timelock being decreased] Decrease timelocks.
- [Timelockable] Set the `performanceFee`.
  The performance fee is capped at 50% of generated interest.
- [Timelockable] Set the `managementFee`.
  The management fee is capped at 5% of assets under management annually.
- [Timelockable] Set the `performanceFeeRecipient`.
- [Timelockable] Set the `managementFeeRecipient`.

>   The timelock on increase timelock should be set to a high value (e.g. 3 weeks) after the vault has been created and initial timelocks have been increased, if any.

Reviewer: ```suggestion
  The timelock of increaseTimelock should be set to a high value (e.g. 3 weeks) after the vault has been created and initial timelocks have been increased, if any.
```

>   The timelock of increaseTimelock should be set to a safe value that gives time to detect mistakes (e.g. 1 day) after the vault has been created and initial timelocks have been set.

Author: ```suggestion
  increaseTimelock should be used carefully, because decreaseTimelock is timelocked with the timelock itself. In particular it is possible to make an action irreversible (which is a feature in itself).
```
